File: desktop operating systems/WirelessDataTransmission_WDTFX/src/sample/Platform/MessageForNotifyAboutCompleteDownloadProcess.java
package sample.Platform;

import com.WDTComponents.AlertInterfaces.ILittleMessage;
import com.WDTComponents.AppOption;
import sampleimagePath.AppInfoKt;

public class MessageForNotifyAboutCompleteDownloadProcess
        implements ILittleMessage
{

    {
        System.loadLibrary("WDTMessageForNotifyAboutCompleteDownloadProcess");
    }

    native void showNativeNotifyMessage(
            String mainTitle, String averageText, String smallText, String mainImagePath
            );

    @Override
    public void showMessage(String strMessage) {
        showNativeNotifyMessage("WDT", strMessage, "Files stay at " +
                AppOption.DIRECTORY_FOR_DOWNLOAD_FILES.getAbsolutePath(), AppInfoKt.ICON);
    }
}

File: android operating system/ui/WirelessDataTransmission_WDT/app/src/main/java/ua/edu/onaft/wirelessdatatransmission_wdt/Configuration/MessageForNotifyAboutCompleteDownloadProcess.kt
package ua.edu.onaft.wirelessdatatransmission_wdt.Configuration

import com.WDTComponents.AlertInterfaces.ILittleMessage

class MessageForNotifyAboutCompleteDownloadProcess: ILittleMessage{

    override fun showMessage(strMessage: String) {
        LittleMessageConfiguration().showMessage(strMessage)
    }

}